java/core/src/main/scala/io/pdal/PackedPoints.scala

```diff
@@ -0,0 +1,66 @@
+package io.pdal
+
+import java.nio.ByteBuffer
+import java.util
+
+import scala.collection.JavaConversions._
+
+/**
+  * PackedPoints abstraction to work with packed point(s) in JVM memory.
+  * SizedDimType contains size and offset for a particular packed point with the current set of dims.
+  **/
+case class PackedPoints(bytes: Array[Byte], dimTypes: util.Map[String, SizedDimType], metadata: String = "", schema: String = "") {
```

Comment:
Probably to rename it to `PointCloud` makes sense, as the current name may cause semantics confusion.